donkeycarManagerFront/src/App.vue

```diff
@@ -1,21 +1,30 @@
 <template>
-  <div id="app">
-    <router-view/>
-  </div>
+  <router-view/>
 </template>
 
 <style>
 
+@import url('https://fonts.googleapis.com/css2?family=Source+Sans+Pro:ital,wght@0,400;1,700&display=swap');
+@import url('https://fonts.googleapis.com/icon?family=Material+Icons');
```

Comment:
Ideally we'd want to avoid using external resources, but Its not really necessary (we should have internet access now)

Reply:
okay

Reply:
it's for icons and idk how to import them in another way